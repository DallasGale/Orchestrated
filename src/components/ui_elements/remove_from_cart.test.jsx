import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import RemoveFromCart from './remove_from_cart'

describe('<RemoveFromCart />', () => {
  test('it renders', () => {
    const Component = render(<RemoveFromCart />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
