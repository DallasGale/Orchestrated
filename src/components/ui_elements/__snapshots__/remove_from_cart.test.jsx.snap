// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RemoveFromCart /> it renders 1`] = `
<button
  class="sc-ifAKCX iySagC"
  type="button"
>
  Remove from cart
</button>
`;
